### PR TITLE
done

### PR DESCRIPTION
**Pull request by eclipseeeeee**
Add main.cpp

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Country{
+	public:
```
Comment (abusimbel1) [inline main.cpp]: Better use struct if you don't have any private methods

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	char name[30];
+	int xl;
+	int xr;
+	int yl;
+	int yr;
+	int cnt;
+	int day;
+} C[25];
+
+class City {
```
Comment (abusimbel1) [inline main.cpp]: Better use struct if you don't have any private methods

Comment (eclipseeeeee) [reply main.cpp]: done

File: main.cpp (hunk 1 above)
Comment (eclipseeeeee) [reply main.cpp]: done

**Pull request merged**